exalearn-ui/src/app/pages/login-page/login-page.component.ts

```diff
@@ -12,8 +12,8 @@ import { UserAuth } from '../../interfaces/interfaces';
 export class LoginPageComponent {
 	constructor(public auth: AuthService) {}
 
-	emailFormControl = new FormControl('', [Validators.required, Validators.email]);
-	passwordFormControl = new FormControl('', [Validators.required]);
+	emailFormControl = new FormControl('adminexa@mailnesia.com', [Validators.required, Validators.email]); //clear formState before prod release
+	passwordFormControl = new FormControl('_Test1234', [Validators.required]); //clear formState before prod release
 
```

Comment:
Are you sure you won't forget to delete them?

Reply:
Please add 
// TODO 
comment here with required changes when this is now needed

Reply:
I won't forget. Added //TODO.